algorithms/exact_change.js

```diff
@@ -0,0 +1,84 @@
+/*
+https://www.freecodecamp.com/challenges/exact-change
+
+Design a cash register drawer function checkCashRegister() that accepts purchase
+ price as the first argument (price), payment as the second argument (cash), and
+  cash-in-drawer (cid) as the third argument.
+
+cid is a 2D array listing available currency.
+
+Return the string "Insufficient Funds" if cash-in-drawer is less than the change
+ due. Return the string "Closed" if cash-in-drawer is equal to the change due.
+
+Otherwise, return change in coin and bills, sorted in highest to lowest order.
+
+
+*/
+
+
+// Create an object which hold the denominations and their values
+var denom = [
+  { name: 'ONE HUNDRED', val: 100.00},
+  { name: 'TWENTY', val: 20.00},
+  { name: 'TEN', val: 10.00},
+  { name: 'FIVE', val: 5.00},
+  { name: 'ONE', val: 1.00},
+  { name: 'QUARTER', val: 0.25},
+  { name: 'DIME', val: 0.10},
+  { name: 'NICKEL', val: 0.05},
+  { name: 'PENNY', val: 0.01}
+];
+
+
+function checkCashRegister(price, cash, cid) {
+  var change = cash - price;
+
+
+  // Transform CID array into drawer object
+  var register = cid.reduce(function(accumulator, current) {
```

Comment:
Nice use of `reduce()`. I'd decompose this and create a separate function for transforming the CID array (single responsibility principle).